src/legacy/core_plugins/kibana/public/management/sections/objects/components/object_view/field.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { PureComponent } from 'react';
+import {
+  EuiFieldNumber,
+  EuiFieldText,
+  EuiFormLabel,
+  EuiSwitch,
+  // @ts-ignore
+  EuiCodeEditor,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { FieldState, FieldType } from '../../types';
+
+interface FieldProps {
+  type: FieldType;
+  name: string;
+  value: any;
+  disabled: boolean;
+  state?: FieldState;
+  onChange: (name: string, state: FieldState) => void;
+}
+
+export class Field extends PureComponent<FieldProps> {
+  render() {
+    const { name } = this.props;
+
+    return (
+      <div className="kuiFormSection">
```

Comment:
Can you remove these legacy `kui` classes? Instead, you can just use the `<EuiFormRow label="">` wrapper around the field.